crates/forge_analyzer/src/definitions.rs

```diff
@@ -2466,6 +2478,26 @@ impl Visit for FunctionCollector<'_> {
                     }
                 }
             }
+            Some(Expr::Object(object_lit)) => {
+                object_lit.props.iter().for_each(|prop| {
+                    if let PropOrSpread::Prop(prop) = prop {
+                        if let Prop::Method(MethodProp { key, function }) = &**prop {
+                            if let Some(key_ident) = key.as_ident() {
+                                if let Some(def_id_function) =
+                                    self.res
+                                        .get_prop_from_ident(self.module, id.clone(), key_ident)
+                                {
+                                    self.handle_function(
+                                        &*function,
+                                        Some(def_id_function.clone()),
```

Comment:
This clone is unnecessary, since `DefId` is `Copy`.

Reply:
Fixed. Thank you!